fastval.go

```diff
@@ -121,8 +121,8 @@ func (val FastVal) IsInt() bool {
 }
 
 func (val FastVal) IsUInt() bool {
-	return val.dataType == IntValue ||
-		val.dataType == JsonIntValue
+	return val.dataType == UintValue ||
```

Comment:
This doesn't seem related to math functions.

Reply:
This has been checked in as a separate fix commit prior to this PR.